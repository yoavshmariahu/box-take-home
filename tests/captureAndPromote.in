k a1
K e5
r c1
S c5
G d5

[G S B P]
[p r b]

move c1 c5 promote
