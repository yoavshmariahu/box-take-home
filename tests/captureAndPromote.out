lower player action: move c1 c5 promote
5 |__|__|+r| G| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|__|__|__|
    a  b  c  d  e

Captures UPPER: G S B P
Captures lower: p r b s

UPPER> 
